index-de.html

```diff
@@ -195,7 +200,7 @@ <h3 id="commit">Commit</h3>
   <p>Ein vierter Fall ist, ein Commit mit <a href="#detached">detached HEAD</a>,
   wird weiter unten erklärt.</p>
 
-  <h3 id="checkout">Checkout</h3>
+  <h3 id="checkout">Checkout (Ausprobieren)</h3>
```

Comment:
Ausstempeln refers to exiting the branch. You're right, Ausstempeln doesn't fit because checkout isn't solely used to switch branches.
l usually use checkout to change the branch, l don't use it to try the other branch but to get another version of something.